### PR TITLE
FOLIO: Extract delivery fulfillment config and translations from #3930

### PR DESCRIPTION
**Pull request by maccabeelevine**
The translations were postponed from 10.1 due to translation freeze.

Also, disabling `allowDelivery = false` in this PR effectively enables delivery since the code default is true.

TODO
- [x] Merge #3930 first.

### CODE REVIEW

**Review by demiankatz — CHANGES_REQUESTED** [review]
@maccabeelevine, #3930 has been merged through to the dev-11.0 branch, so you're free to reconcile any differences required here.

**Comment by maccabeelevine** [comment]
> @maccabeelevine, #3930 has been merged through to the dev-11.0 branch, so you're free to reconcile any differences required here.

Darn, I resolved the differences but accidentally merged from dev instead of dev-11.0.  This is why I hate dev branches :)  Please help if you can, or I will muddle it out.

**Comment by demiankatz** [comment]
> Darn, I resolved the differences but accidentally merged from dev instead of dev-11.0. This is why I hate dev branches :) Please help if you can, or I will muddle it out.

I think I've gotten it straightened out! Just a matter of making sure dev was merged into dev-11.0 and then merging dev-11.0 into here. If it looks good to you, I'll review further next week!

**Comment by maccabeelevine** [comment]
> I think I've gotten it straightened out! Just a matter of making sure dev was merged into dev-11.0 and then merging dev-11.0 into here. If it looks good to you, I'll review further next week!

Yes, looks good!  Thanks for the save!

File: languages/en.ini [hunk 1]
```diff
@@ -1226,6 +1228,8 @@ renew_success = "Renewal Successful"
 renew_success_summary = "Successfully renewed {count, plural, =1 {1 item} other {# items}}."
 Renewed = "Renewed"
 Request full text = "Request full text"
+request_group_fulfillment_method_delivery = "Delivery"
```
Comment (demiankatz) [inline languages/en.ini:1231]: I'm probably missing some but I don't see where/how these request_group_fulfillment_method* strings are used. Do we still need them for something?

**Review by demiankatz — CHANGES_REQUESTED** [review]
Just one question before I merge this....

File: languages/en.ini (hunk 1 above)
Comment (maccabeelevine) [reply languages/en.ini:1231]: Yes, they are used as 'name' params [in the Folio driver](https://github.com/vufind-org/vufind/blob/dev-11.0/module/VuFind/src/VuFind/ILS/Driver/Folio.php#L1623) minus the `request_group_` prefix, which is added in [hold.phtml](https://github.com/vufind-org/vufind/blob/dev-11.0/themes/bootstrap3/templates/record/hold.phtml#L64) to output the request group name.  Hard to search for!

Comment (demiankatz) [reply languages/en.ini:1231]: Ahh, tricky! Thanks for the clarification. :-)

**Review by demiankatz — APPROVED** [review]

**Pull request merged**